pages/common/msmtp.md

```diff
@@ -0,0 +1,16 @@
+# msmtp
+
+> An SMTP client.
+> In the default sendmail mode, msmtp reads a mail from standard input and sendscit to an SMTP server for delivery.
+
+- Send a mail using the default account configured in `~/.msmtprc`:
```

Comment:
"send a mail" sounds a little awkward, but maybe it's my ear. I'd have used "Send an email".